src/mango/test/mango.py

```diff
@@ -156,13 +157,29 @@ def create_index(
         assert r.json()["name"] is not None
 
         created = r.json()["result"] == "created"
-        if created:
-            # wait until the database reports the index as available
-            while len(self.get_index(r.json()["id"], r.json()["name"])) < 1:
-                delay(t=0.1)
+        if created and wait_for_built_index:
+            # wait until the database reports the index as available and build
+            while (
```

Comment:
How about:

```python
while True:
    idx = self.get_index(r.json()["id"], r.json()["name"])[0]
    if idx["build_status"] == "ready":
        break
    delay(t = 0.2)
```

Reply:
This is the formatting python-black recommended